packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -110,39 +121,47 @@ export class WaferMap extends FoundationElement {
      */
     public render(): void {
         this.renderQueued = false;
-        if (
-            this.canvasSideLength === undefined
-            || this.canvasSideLength === 0
-        ) {
+        this.initalizeInternalModules();
+        this.renderer?.drawWafer();
+    }
+
+    /**
+     * @internal
+     */
+    public queueRender(): void {
```

Comment:
This is only used inside this file and can be made private

Reply:
applied